bitcoin/script/miniscript.h

```diff
@@ -148,6 +162,11 @@ inline constexpr Type operator"" _mst(const char* c, size_t l) {
         *c == 's' ? 1 << 11 : // Safe property
         *c == 'm' ? 1 << 12 : // Nonmalleable property
         *c == 'x' ? 1 << 13 : // Expensive verify
+        *c == 'g' ? 1 << 14 : // older: contains relative time timelock   (csv_time)
+        *c == 'h' ? 1 << 15 : // older: contains relative height timelock (csv_height)
+        *c == 'i' ? 1 << 16 : // after: contains time timelock   (cltv_time)
+        *c == 'j' ? 1 << 17 : // after: contains height timelock   (cltv_height)
+        *c == 'k' ? 1 << 18 : // contains a combination of height and time
```

Comment:
I feel that using the negation of this (does not contain a combination of height and time) would be more in line with the existing properties (which are also "positive" ones, like safeness/nonmalleability). Then the compiler can be requested to select for that property, rather than needing to exclude it separately.

Reply:
Agreed

Reply:
Done